oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -211,11 +211,31 @@ static FullGCMode getFullGcMode() {
         return fullGcMode;
     }
 
+    /**
+     * Set the full GC mode to be used according to the provided configuration value.
+     * The configuration value will be ignored if it is set to any other values than the supported ones.
+     * @param fullGcModeConfig
+     */
+    static void setFullGcMode(int fullGcModeConfig) {
+        switch (fullGcModeConfig) {
+            case 0:
+                fullGcMode = NONE;
+                break;
+            case 2:
+                fullGcMode = GAP_ORPHANS;
+                break;
+            case 3:
+                fullGcMode = GAP_ORPHANS_EMPTYPROPS;
+                break;
+        }
+    }
```

Comment:
I would suggest to add a `default` catch all section here that logs the fact that a not support mode was used and that the default mode is used instead

Reply:
Done, thank you for the suggestion.